### PR TITLE
[1LP][WIPTEST] apply the six iteritems fixup of python-modernize

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6051a3d9e7d043d55b376d81020356f211c42a36:

`cfme/tests/containers/test_logging.py`:
- :bangbang: Line 36:101: **E501** *line too long (103 > 100 characters)*

`cfme/tests/containers/test_table_views.py`:
- :bangbang: Line 44:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 47:101: **E501** *line too long (103 > 100 characters)*

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 1869:101: **E501** *line too long (102 > 100 characters)*

`cfme/utils/perf_message_stats.py`:
- :bangbang: Line 356:101: **E501** *line too long (106 > 100 characters)*

`cfme/utils/release.py`:
- :bangbang: Line 101:101: **E501** *line too long (101 > 100 characters)*

`scripts/sync_template_tracker.py`:
- :bangbang: Line 127:101: **E501** *line too long (103 > 100 characters)*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: scripts/coverage_result.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python2
 from __future__ import print_function
+from __future__ import absolute_import
```
Comment (quarckster) [inline scripts/coverage_result.py]: **OPTIONAL**
And here.

File: scripts/apishow.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 """
 from __future__ import print_function
 
+from __future__ import absolute_import
```
Comment (quarckster) [inline scripts/apishow.py]: **OPTIONAL**
We can do slightly better here.

File: scripts/ipappliance.py [hunk 3]
```diff
@@ -20,6 +20,7 @@
 
 """
 from __future__ import print_function
+from __future__ import absolute_import
```
Comment (quarckster) [inline scripts/ipappliance.py]: **OPTIONAL**
And here.

**Comment by psav** [comment]
Please fix up the description

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
Setting WIPTEST to remove this from filters since it has NOMERGE set.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b9a562a1e18f4970afd4e2e8f592312095028d74:

`cfme/tests/containers/test_table_views.py`:
- :bangbang: Line 46:8: **E999** *IndentationError: unexpected indent*
- :bangbang: Line 46:9: **E113** *unexpected indentation*
- :bangbang: Line 52:9: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 53:5: **E122** *continuation line missing indentation or outdented*

`cfme/tests/test_db_migrate.py`:
- :red_circle: Line 17:1: **F401** *'six' imported but unused*
- :red_circle: Line 72:1: **F811** *redefinition of unused 'temp_appliance_extended_db' from line 43*
- :red_circle: Line 81:1: **F811** *redefinition of unused 'temp_appliance_remote' from line 52*
- :red_circle: Line 91:1: **F811** *redefinition of unused 'temp_appliance_global_region' from line 62*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
closing due to repeat rebase issues

**Pull request closed without merging**